pkg/podmanager/fdb_process_group.go

```diff
@@ -34,7 +34,7 @@ import (
 var processIDRegex = regexp.MustCompile(`^([\w-]+-\d)-\d$`)
 
 // ParseProcessGroupID extracts the components of a process group ID.
-func ParseProcessGroupID(id string) (fdbv1beta2.ProcessClass, int, error) {
+func ParseProcessGroupID(id string) (int, error) {
```

Comment:
I know that's a breaking change but the underlying method returned actually the prefix and not the process class, which means the signature was wrong. We also never used the prefix somewhere and only make use of the ID number.

Reply:
We should change this back and make a note at the top that the process class is actually a string.

Reply:
We should deprecate it as well, if we have no need identified for the version that returns a process class or prefix.